jaxrs-api/src/main/java/jakarta/ws/rs/container/CompletionCallback.java

```diff
@@ -30,19 +30,19 @@
 public interface CompletionCallback {
     /**
      * A completion callback notification method that will be invoked when the request processing is finished, after a
-     * response is processed and is sent back to the client or when an unmapped throwable has been propagated to the hosting
-     * I/O container.
+     * response is processed and is sent back to the client or when an unmapped throwable has been propagated to the default
+     * exception mapper.
      * <p>
```

Comment:
Same comment as above about "default exception mapper". And some more occurrences below as well.